sql/parser/eval.go

```diff
@@ -316,6 +319,13 @@ var binOps = map[binArgs]binOp{
 	},
 }
 
+func inDays(d time.Duration) (DDate, error) {
+	if d%(24*time.Hour) != 0 {
```

Comment:
Ah, right. Because days are not always 24 hours (daylight savings). Which means that the result of `inDays` cannot be used in an operation on any time that is not already in UTC. It would be better to make this `addDays(time.Time, time.Duration)` which would either assert that the given time is in UTC or do the right thing in the given time's time zone.